shared/common-adapters/inline-banner.js

```diff
@@ -0,0 +1,46 @@
+// @flow
+import * as React from 'react'
+import Box2 from './box'
+import Text, {type Background} from './text'
+import * as Styles from '../styles'
+
+// Show a banner inline in a layout
+//
+// This adapter is a WIP as of 11/1/2018, add props and functionality as new
```

Comment:
i don't think this comment adds much value, this is true of everything

Reply:
I can remove it. I thought we typically wouldn't want to add things to common adapters except as a last resort, so this comment could overrule that until it's baked a while longer.

Reply:
i think we likely don't want this as a common then